### DISCUSSION TITLE
Nothing to display if I click the 'Get' button when the wallet has already been installed

### DISCUSSION THREAD

**Discussion by KurtneySh** [rainbow-me/rainbowkit#2116] [Q&A]
Is it supposed to be this way or is it just a bug?

https://github.com/user-attachments/assets/4ae0ceea-25cc-47be-8f93-60c24583d8cd

**Comment by 0xmkh** [comment]
@KurtneySh Thanks for finding the bug! This should not happen so i've gone ahead and raised an issue here https://github.com/rainbow-me/rainbowkit/issues/2121
